eng/liveBuilds.targets

```diff
@@ -78,10 +78,11 @@
     <CoreCLRCrossTargetComponentDirName Condition="'$(TargetArchitecture)' == 'arm' and '$(BuildArchitecture)' != 'arm' and '$(TargetsLinux)' == 'true'">x64</CoreCLRCrossTargetComponentDirName>
   </PropertyGroup>
 
-  <Target Name="ResolveCoreCLRFilesFromLocalBuild">
-    <Error Condition="!Exists('$(CoreCLRArtifactsPath)')" Text="The CoreCLR artifacts path does not exist '$(CoreCLRArtifactsPath)'. The CoreCLR subset category must be built before building this project." />
+  <Target Name="ResolveVMFilesFromLocalBuild">
+    <Error Condition="!Exists('$(CoreCLRArtifactsPath)') and '$(RuntimeFlavor)' == 'CoreCLR'" Text="The CoreCLR artifacts path does not exist '$(CoreCLRArtifactsPath)'. The CoreCLR subset category must be built before building this project." />
+    <Error Condition="!Exists('$(MonoArtifactsPath)') and '$(RuntimeFlavor)' == 'Mono'" Text="The Mono artifacts path does not exist '$(MonoArtifactsPath)'. The Mono subset category must be built before building this project." />
 
-    <PropertyGroup>
+    <PropertyGroup Condition="'$(RuntimeFlavor)' == 'CoreCLR'">
```

Comment:
Would it be clearer if we group the Property/Item groups by their conditions?
```
<Choose>
   <When Condition="'$(RuntimeFlavor)' == 'CoreCLR'">
      <PropertyGroup></PropertyGroup>
      <ItemGroup></ItemGroup>
   </When>
   <When Condition="'$(RuntimeFlavor)' == 'Mono'">
      <PropertyGroup></PropertyGroup>
      <ItemGroup></ItemGroup>
   </When>
</Choose>
```

Reply:
It looks like `<Choose>` only works at the project level, not inside a Target: https://github.com/Microsoft/msbuild/issues/2056